### ISSUE TITLE
downloads fails when bandwith is poor/low

### ISSUE THREAD

**Issue by ArnaudCazin** [mayth/go-simple-upload-server#44]
Hi, 

download a file fails when  bandwith is poor / low 
i can reproduce when limiting bw with curl 
for example : 
```
❯ curl  --limit-rate 1200k  https://needed.xxx.xxx/files/ffmpeg_6_1 -H "Authorization: Bearer XXX" -o /tmp/ffmpeg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 34 75.2M   34 25.6M    0     0  1147k      0  0:01:07  0:00:22  0:00:45  967k
curl: (92) HTTP/2 stream 1 was not closed cleanly: PROTOCOL_ERROR (err 1)
```
the error: 
curl: (92) HTTP/2 stream 1 was not closed cleanly: PROTOCOL_ERROR (err 1)

the file size to download is 75mb (not so large)

Regards

**Comment by mayth** [comment]
Thank you for your report.
I reproduced this issue. It seems that this is caused because `WriteTimeout` is fixed to 15 seconds. Since this configuration,  the app closes the connection after 15 seconds.
https://github.com/mayth/go-simple-upload-server/blob/55c599582ca4ee02bdd89c026f2057a4eb24b3fc/pkg/server.go#L104
I will fix this issue. Just a moment please.

**Comment by mayth** [comment]
@ArnaudCazin 
This problem is fixed in #45 v2.1.0.
Now, `WriteTimeout` is set to zero, means no timeout, by default.
Note that `ReadTimeout` (upload time limit) still defaults to 15s to avoid too many connections being held. This might cause the similar issue on uploading large files. If you encounter such a situation, consider increasing read timeout.

If the problem is not solved, feel free to reopen this issue.

**Comment by ArnaudCazin** [comment]
Great , it's working fine 
thanks a lot
